hoomd/md/test/test_MolecularForceCompute.cc

```diff
@@ -157,7 +157,7 @@ void comparison_test(std::shared_ptr<ExecutionConfiguration> exec_conf_cpu, std:
         for (unsigned int j = 0; j < nptl; ++j)
             {
             // choose a molecule tag 0<=mol_tag< nptl
-            unsigned int t = saru.f()*(nptl+1);
+            unsigned int t = hoomd::detail::UniformIntDistribution(nptl+1)(rng);
```

Comment:
Based on the bounds in the comment above, should this be `nptl-1`? But, I also don't understand how the original code matches the comment either.

Reply:
yeah, I believe the comment is wrong, though `nptl+1` seems correct (see line 161).

Reply:
Based on the code, it seems that this should be correct.
```
            // choose a molecule tag 0 <= mol_tag <= nptl
            unsigned int t = hoomd::detail::UniformIntDistribution(nptl)(rng);
```
It would be an error to assign a molecule tag of `nptl + 1` and `UniformIntDistribution` returns values inclusive of the maximum value given.

Reply:
> Based on the code, it seems that this should be correct.
> 
> ```
>             // choose a molecule tag 0 <= mol_tag <= nptl
>             unsigned int t = hoomd::detail::UniformIntDistribution(nptl)(rng);
> ```
> It would be an error to assign a molecule tag of `nptl + 1` and `UniformIntDistribution` returns values inclusive of the maximum value given.

I overlooked it was inclusive. In this case, yes, I agree.